hw03_wange2.py

```diff
@@ -0,0 +1,73 @@
+# Name: Eun Sung Wang
+# Username: wange2
+# Google Doc: https://docs.google.com/document/d/1jl7uXjys2F0Tu1IFPvjZcEspKPECxvctfCDyoPJD6h8/edit?usp=sharing
+
+import turtle
+
```

Comment:
Check for gray squigglies. They indicate style standards, such as 2 blank lines before all function definitions.

Reply:
I have fixed them.